tree_segment.py

```diff
@@ -0,0 +1,434 @@
+import laspy
+import matplotlib.pyplot as plt
+import math
+import numpy as np
+import random
+import time
+import pandas as pd
+import argparse
+import re
+import os
+
+
+class Point_Cloud:
+    # object that stores the points in a point cloud, initialized from a las file
+    # points can be enabled or disabled, disabled points are ignored when searching for stems
+    def __init__(self, file_name, verbose=False):
+        all_data = laspy.read(file_name)
+        height_of_points_above_ground = np.array(
```

Comment:
Consider moving this computation into the "Point" class.

Reply:
What computation do you mean? Mostly this is just reading the laspy file and accessing two columns (xyz and points.array[-1]).